backend/src/core/models/steam.apps.aggregate.js

```diff
@@ -9,30 +9,30 @@ export class SteamAppsAggregate {
   }
 
   get content() {
-    return this.#apps;
+    const contentCopy = new SteamAppsAggregate(this.#apps).#apps;
```

Comment:
This works for now but won't anymore when a new property with an array as type comes and is not properly copied. Please use `deepCopy` instead as you should above.